### PR TITLE
Enable a way to inject Hubspot Tether modules into ember-tether.

### PR DESCRIPTION
**Pull request by keppy**
I needed to use tether modules, which need to be loaded into Tether. I have enabled the option of passing in a `modules` property to the `ember-tether` component. This allows anyone using ember-tether to pass in an existing hubspot tether module without needing to have an initializer or other ugly code.
cheers

NOTE: Needs merged first https://github.com/yapplabs/ember-tether/pull/20 so that we can get the tests to pass in this branch.

### CODE REVIEW

**Comment by chrislopresto** [comment]
@keppy Want to rebase now that we merged #20?

**Comment by keppy** [comment]
@chrislopresto will do, thx

File: addon/modules/transition-module.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
+let transitionModule = {
+  className: 'transition',
+  thresholdX: 50,
+  thresholdY: 50,
+
+  position(props) {
+    let className = `${this.options.classPrefix}-${transitionModule.className}`;
+    console.log(className);
```
Comment (kiwiupover) [inline addon/modules/transition-module.js]: Should there be a `console.log` here?

Comment (keppy) [inline addon/modules/transition-module.js]: nope, thanks.

**Comment by chrislopresto** [comment]
Actually, I asked for the rebase before taking a close enough look. Since modules are pushed on to the `Tether` global, I don't think it makes sense to scope module setup at an `ember-tether` instance level.

There are a few other issues with the PR.. for example, looks like some extraneous files are leftover in the addon directory. But we can hold off on those for now.

@keppy What is your reasoning against using, let's say, an initializer in your app to push on to the `Tether.modules` array once?

**Comment by chrislopresto** [comment]
@keppy I'm happy to chat in the Ember Community Slack about any of this, too.

**Comment by chrislopresto** [comment]
I'm going to close this, as I don't believe there is any blocker to configuring tether modules. 

@keppy Feel free to reach out to me here or in the Ember Community Slack if you want to discuss ways this addon could facilitate tether module configuration.

**Pull request closed without merging**